### PR TITLE
enableable modifier changes

### PR DESCRIPTION
**Pull request by NikhilSuresh24**
Fixed master for changes of enableable modifier

### CODE REVIEW

File: src/org/usfirst/frc4904/robot/humaninterface/drivers/AlignAssist.java [hunk 1]
```diff
@@ -15,8 +15,8 @@
 	public AlignAssist(GearAlign gearAlign, EnableableModifier fineModifier) {
 		this.gearAlign = gearAlign;
 		this.fineModifier = fineModifier;
-		enableFineModifier = new SetEnableableModifier(fineModifier, true);
-		disableFineModifier = new SetEnableableModifier(fineModifier, false);
+		enableFineModifier = new SetEnableableModifier(true, fineModifier);
```
Comment (billwpierce) [inline src/org/usfirst/frc4904/robot/humaninterface/drivers/AlignAssist.java]: Is this change correct? The construction of SetEnableableModifier takes in the modifier and then a boolean. Plz explain.

File: src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java [hunk 2]
```diff
@@ -47,8 +47,8 @@ public void bindCommands() {
 		RobotMap.Component.driverXbox.lb
 			.onlyWhileHeld(
 				new RunAllSequential(new WaitCommand("Third gear activation delay", NathanGain.THIRD_GEAR_ENGAGE_DELAY_SECONDS),
-					new SetEnableableModifier(modifier, true)));
-		RobotMap.Component.driverXbox.lb.whenReleased(new SetEnableableModifier(modifier, true));
+					new SetEnableableModifier(true, modifier)));
+		RobotMap.Component.driverXbox.lb.whenReleased(new SetEnableableModifier(true, modifier));
```
Comment (billwpierce) [inline src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java]: Same as above

**Review by billwpierce — COMMENTED** [review]
I might be misunderstanding, but please explain why you changed this in the description of this.

File: src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java (hunk 2 above)
Comment (NikhilSuresh24) [reply src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java]: standard update

File: src/org/usfirst/frc4904/robot/humaninterface/drivers/AlignAssist.java (hunk 1 above)
Comment (NikhilSuresh24) [reply src/org/usfirst/frc4904/robot/humaninterface/drivers/AlignAssist.java]: standard update

**Review by billwpierce — APPROVED** [review]
My previous comments were wrong.

**Review by ajnadel — APPROVED** [review]
If the other standard changes are tested, then I'm definitely okay with this!

**Comment by c0d3rman** [comment]
This made redundant by #30.

**Pull request closed without merging**